stack.yaml

```diff
@@ -2,6 +2,7 @@ flags: {}
 extra-package-dbs: []
 packages:
 - .
-extra-deps: []
+extra-deps:
+  - ansi-terminal-1.0
```

Comment:
Why did you add this dep here? It should be already part of the build-depends in the cabal file

Reply:
I wanted to make sure it works with `ansi-terminal-1.0`, but it is not in the latest lts yet: https://www.stackage.org/package/ansi-terminal

Another choice was to switch it to nightly.

Matters not to me, let me know how you'd like it.